src/aavruti/EmpManagerWebTable.java

```diff
@@ -0,0 +1,94 @@
+package aavruti;
+
+import aavruti.base.PredefinedActions;
+
+import java.util.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class EmpManagerWebTable extends PredefinedActions{
+
+	WebDriver driver;
+
+	void launchBrowser() {
+		driver = start();
+	}
+
+	void navigateToDemoTable() throws InterruptedException{
+		driver.findElement(By.xpath("//a[text()='Demo Tables']")).click();	
+		Thread.sleep(3000);
+	}
+
+	//Find how many employees reporting to each manager ID.
+	HashMap<String,Integer> findCountOfEmpReportingToManager() {
+
+		HashMap<String,Integer> totalEmpPerManager = new HashMap<>();
+
+		int totalRows = driver.findElements(By.xpath("//table[@class='table table-striped']/tbody/tr")).size();
+		String empID = ",";
+
+		for(int index=1;index<=totalRows;index++) {
+			String managerID = driver.findElement(By.xpath("//table[@class='table table-striped']/tbody/tr["+index+"]/td[4]")).getText();
+			String empIDDuplicate = driver.findElement(By.xpath("//table[@class='table table-striped']/tbody/tr["+index+"]/td[2]")).getText();
+
+			if(totalEmpPerManager.containsKey(managerID)) {
+				if(!empID.contains("," + empIDDuplicate + ","))				
```

Comment:
1. Not getting this condition use on adding empID

Reply:
@harshvegada :- 
There were two entries of same Employee 'Sumit Mate' in table and same employee was getting count two times for same ManagerID. So i thought to add all EmpID one by one in empID variable and check the condition that if same Employee comes again it will not get count. As you can see in below screenshot. 

Please let me know if it is not correct.

![image](https://user-images.githubusercontent.com/66829349/91209710-c1d72800-e729-11ea-9efb-39ccb867f056.png)

Reply:
okay, got it,
I missed your else condition where you concating empID, that why i got confused for this condition
now m clear,
Thanks for clarification.